src/main/java/frc/trigon/robot/subsystems/elevator/simulationelevator/SimulationElevatorConstants.java

```diff
@@ -0,0 +1,61 @@
+package frc.trigon.robot.subsystems.elevator.simulationelevator;
+
+import edu.wpi.first.math.system.plant.DCMotor;
+import frc.trigon.robot.simulation.ElevatorSimulation;
+import frc.trigon.robot.simulation.MotorSimulationConfiguration;
+import frc.trigon.robot.subsystems.elevator.ElevatorConstants;
+
+public class SimulationElevatorConstants {
+    static final double VOLTAGE_COMPENSATION_SATURATION = 12;
+    static final boolean FOC_ENABLED = true;
+    private static final int NUMBER_OF_MOTORS = 2;
+    private static final double
+            GEAR_RATIO = 1,
```

Comment:
this is configured in both the real and sim IOs, so just put it in ElevatorConstants